.env.example

```
@@ -0,0 +1,48 @@
+# Eatsfine Backend - 환경변수 설정 가이드
+# ============================================
+# 이 파일을 `.env`로 복사한 뒤, 실제 값을 입력하세요.
+# cp .env.example .env
+#
+# ⚠️ 주의: .env 파일은 절대 Git에 커밋하지 마세요!
+# ============================================
+
+# ── Database (MySQL) ──────────────────────────
+DB_HOST=localhost
+DB_PORT=3306
+DB_NAME=eatsfine_local
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_PORT key

(UnorderedKey)